find_target_name_from_header.py

```diff
@@ -137,34 +107,42 @@ def query_simbad(date,coords,name=None,limit_G_mag=15,metadata=None,verbose=True
 
     if name is not None:
         search = customSimbad.query_object(name)
-        validSearch = search[search['FLUX_G']<limit_G_mag]
-        nb_stars = len(validSearch)                
+        
+        if type(search)==type(None): # a binary system component may not be identified by Simbad()
+            if (name[-1]=='A') | (name[-1]=='B'): name=name[:-1]
+            search = customSimbad.query_object(name)
+            
+        if type(search)!=type(None):
+            filter_cols = ['FLUX_G', 'FLUX_J', 'FLUX_H', 'FLUX_K', 'FLUX_U', 'FLUX_B', 'FLUX_V', 'FLUX_R', 'FLUX_I']
```

Comment:
Thanks for adding this check incase the G mag is not available.
However I would not use the K band mag here, because a star can have K<15 and G>15 but this cannot be the target in this case as the AO cannot close the loop on a star with visible mag > 15. I think it is still OK to keep the J and H though, but the K. 

I propose also to use the following order to make the test quicker as the V band is the most common:
['FLUX_G', 'FLUX_V', 'FLUX_R','FLUX_U', 'FLUX_B','FLUX_I','FLUX_J', 'FLUX_H']

Reply:
Okay, I have fixed the line accordingly.